internal/runtime/internal/controller/value_cache.go

```diff
@@ -234,3 +239,26 @@ func (vc *valueCache) buildValue(from []ComponentID, offset int) interface{} {
 	}
 	return attrs
 }
+
+// deepMergeMaps merges two maps. It uses the values of map2 in case of conflict.
```

Comment:
So in this case it uses the child var with the parent var in case of a conflict?

Reply:
yes, not sure if conflicts should be allowed at all though. I will give another go at this by changing the value_cache a bit more